Our Values page

charlisable opened the pull request:
Fixed all accessibility issues and just needs the header and footer. The page is ready for review.

TerryHCollins committed:
Merge branch 'development' of https://github.com/indigenousfriends/itlct-website-redesign into ourvalues-page

TerryHCollins committed:
Merge branch 'development' of https://github.com/indigenousfriends/itlct-website-redesign into ourvalues-page

TerryHCollins committed:
Capitalized Indigenous and Native in html

TerryHCollins committed:
Merge branch 'development' of https://github.com/indigenousfriends/itlct-website-redesign into ourvalues-page

TerryHCollins committed:
Merge branch 'ourvalues-page' of https://github.com/indigenousfriends/itlct-website-redesign into ourvalues-page

TerryHCollins committed:
Merge branch 'development' of https://github.com/indigenousfriends/itlct-website-redesign into ourvalues-page

TerryHCollins committed:
Merge branch 'ourvalues-page' of https://github.com/indigenousfriends/itlct-website-redesign into ourvalues-page

TerryHCollins committed:
Merge branch 'ourvalues-page' of https://github.com/indigenousfriends/itlct-website-redesign into ourvalues-page

TerryHCollins committed:
Merge branch 'development' of https://github.com/indigenousfriends/itlct-website-redesign into ourvalues-page

TerryHCollins committed:
Capitalization on Native and Indigenous, added top margin to WE SUPPORT in mobile

TerryHCollins committed:
updated mobile h1 font specifics, desktop support logos padding bottom added

charlisable committed:
added width and height to hero section

charlisable committed:
Merge branch 'development' into ourvalues-page

charlisable committed:
fixed spacing issues on empowering and support us section

charlisable committed:
fixed spacing issues with card text, fixed red block position

charlisable committed:
replaced code with variables, removed dot in front of image paths

charlisable committed:
Merge branch 'development' into ourvalues-page

charlisable committed:
added updated header and footer

charlisable committed:
hid mobile hero image in desktop

charlisable committed:
fixed heading issue and removed p tag

MaidenManitoba commented on css/values.css:
```diff
@@ -106,14 +111,11 @@ Page Component Styles (icon grid, team member card)
-.support-header,
-.support-text {
-	width: var(--desktop-wrapper-width);
-	margin-left: 20px;
```
You can use a spacing variable here?

MaidenManitoba reviewed (commented):
<img width="474" alt="Screenshot 2024-01-30 113144" src="https://github.com/indigenousfriends/itlct-website-redesign/assets/97715061/8d200a62-0b15-4608-a2c4-6f06a3b9db12">
There is some spacing issues in the We support section. The font is different. The font colour is supposed to be Red for the We Support heading. The technology and innovation card needs to be changed to just Technology for mobile. Other than these small changes, it looks great!

samifa308 reviewed (commented):
its looking good, just a few minor changes!

charlisable committed:
Merge branch 'development' into ourvalues-page

charlisable committed:
added updated footer, fixed spacing issues in the Support section

charlisable commented:
Fixed spacing issues in the Support section, and removed extra text in mobile for Technology and Innovation title. There is a contrast issue when the text is red so I kept it white for now. Also added the updated footer.

liannarice commented:
this looks good but is it confusing to anyone else with the where to click for the links for UNDRIP? I was clicking the icons before I realized that the link was within the text.

charlisable committed:
fixed line height for empowering section

charlisable committed:
Merge branch 'development' into ourvalues-page

maxharrisnet commented on values.html:
```diff
@@ -265,27 +235,27 @@ <h2 class="h2-styles">WE SUPPORT</h2>
 				</svg>
 			</div>
-			<div class="support-text">
+			<div class="support-text wrapper">
```
The heading text in the UN section is too wide. It needs a max-width on desktop so that it drops to the second line sooner. 

<img width="1404" alt="Screen Shot 2024-01-30 at 10 32 52 AM" src="https://github.com/indigenousfriends/itlct-website-redesign/assets/704697/4526c126-c989-47b2-b283-7e1abf786940">

charlisable replied:
Added max-width to .support p so the text goes to the second line for desktop

maxharrisnet reviewed (changes requested):
This is almost done. If we can get the white text in the UN section looking right on desktop then it's good to go!

charlisable committed:
added max-width to text on We Support section for desktop

charlisable committed:
Merge branch 'development' into ourvalues-page

maxharrisnet reviewed (approved):
Looks great!
